sandcastle/api.py

```diff
@@ -616,7 +620,12 @@ def _copy_path_from_pod(self, local_dir: Path, pod_dir: Path):
             )
             tar_cmd = f"tar -czf {remote_tar_path} -C {pod_dir} $({grc})"
             pack_cmd = ["bash", "-c", tar_cmd]
-            self._do_exec(pack_cmd)
+            output = self._do_exec(pack_cmd)
+            if output:
+                logger.debug(f"output from tar -czf: {output!r}")
+            # let's make sure the archive is present
+            output = self._do_exec(["ls", "-lh", str(remote_tar_path)])
```

Comment:
OK, now I get why there is another `tar` in the logs...

Reply:
this will be the new error message probably - the archive is not present in the sandbox

but at the same time, we'll have output of tar -czf from the above

Reply:
@TomasTomecek but I was confused that this is not the archive we are creating for the SRPM...

Reply:
nope, this is our internal bullshit :)

Reply:
But very misleading since we have problems with archives and their paths very often...;)